11ty/es/index.njk

```diff
@@ -0,0 +1,26 @@
+---
+layout: layouts/index.njk
+---
+
+<header class="auto-grid introduction">
+  <h1 class="introduction__title thicc-headline" id="title">
+    Self-Defined
+  </h1>
+  <p class="introduction__body">Un diccionario moderno sobre nosotres.<br>Definimos nuestras palabras, pero ellas no nos definen.</p>
+  <div>
+    <p class="summary">
+      Self-Defined busca proporcionar definiciones más inclusivas, holísticas y fluidas para reflejar las diversas perspectivas del mundo moderno.
+    </p>
+    <p>
+      Con base en el vocabulario, podemos comenzar a entender experiencias vividas de personas diferentes a nosotres. Las palabras pueden proporcionar un sentido de identidad y nos permiten encontrar similitudes a través de experiencias comunes.
+    </p>
+  </div>
+  <div class="box">
+    <h2 class="sub-headline">Formas de ayudar</h2>
+    <ol class="list">
+      <li>Envía palabras y definiciones o contribuye a nuestra base de código  a través de <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferrer"><i>pull requests</i></a> o <a href="https://github.com/tatianamac/selfdefined/issues" rel="noreferrer"><i>issues</i></a>. Comienza con nuestras <a href="https://github.com/selfdefined/web-app/blob/prod/CONTRIBUTING.md" rel="noreferrer">directrices de contribución</a> y la <a href="/documentation/">documentación</a>.</li>
```

Comment:
This is super minor, but there is a double space in `código  a` which we might want to remove :)